src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -96,7 +96,10 @@ public <T extends Command, U extends SchemaObject> Uni<CommandResult> processCom
               }
               // But other exception types are unexpected, so log for now
               logger.warn(
-                  "Command '{}' failed with exception", command.getClass().getSimpleName(), t);
+                  String.format(
+                      "Command '%s' failed with %s: %s",
+                      command.getClass().getSimpleName(), t.getClass().getName(), t.getMessage()),
```

Comment:
`t.getMessage()` might return `null` in some cases which might lead to reading this log as something is null.
For example:
It might log
`Command CreateCollection failed with IllegalArgumentException: null` 

Should we print `t.getMessage()` only if its not null?

Reply:
I can change `null` to something else, good idea

Reply:
@kathirsvn changed, does this look better?

Reply:
LGTM